src/protocol.h

```diff
@@ -425,7 +425,7 @@ class CInv
     // Combined-message helper methods
     bool IsGenTxMsg()     const { return type == MSG_TX || type == MSG_WTX || type == MSG_WITNESS_TX; }
 
-    int type;
+    uint32_t type;
```

Comment:
ACK on using a sized type here.
However, changing from signed to unsigned is, in principle, a P2P protocol change.
This needs to be carefully reviewed for potential unexpected by-effects.